hydralit_components/Experimental/experiments.py

```diff
@@ -81,7 +81,9 @@ def _add_injections(new_scripts,new_style_sheets,custom_html,inject_raw_script):
             hidden_header.append(new_script)
 
         if inject_raw_script is not None:           
-            hidden_header.append(soup.new_tag('script').append(inject_raw_script))

```

Comment:
the outer `hidden_header.append` would be passed a `None` since  the append in `soup.new_tag('script').append(inject_raw_script)` returns nothing (None).